markdown/api-docs/cart-and-checkout/embedded-checkout.md

```diff
@@ -0,0 +1,335 @@
+    
+<!-- theme:  -->
+
+For products without options, you only need to send `quantity` and `product_id` in a `line_items` object to create a cart.
+
+</div>
+</div>
+</div>
+
+<!--
+title: "Create Cart POST"
+subtitle: ""
+lineNumbers: true
+-->
+
+**Create Cart POST**
+
+```js
+{
+  "channel_id": 15001,
+  "line_items": [
+    {
+      "quantity": 1,
+      "product_id": 80,
+      "variant_id": 64
+    }
+  ]
+}
+
+```
+
+This will return a UUID format `id` that you will use as the `cart_id`.
+
+<a href='#embedded-checkout_create-routes' aria-hidden='true' class='block-anchor'  id='embedded-checkout_create-routes'><i aria-hidden='true' class='linkify icon'></i></a>
+
+## 5: Create Routes
+
+Next we'll create routes to redirect customers from links on your external site to the appropriate BigCommerce URLs.
+
+To do so, make a PUT to this endpoint: `https://api.bigcommerce.com/stores/{{store_hash}}/v3/sites/{{site_id}}/routes`
+
+<!--
+title: "Create Route PUT"
+subtitle: ""
+lineNumbers: true
+-->
+
+**Create Cart POST**
+
+```js
+[
+  {
+    type: "product",
+    matching: "*",
+    route: "/products/"
+  }
+];
+```
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--">
+<div class="HubBlock-content">
+    
+<!-- theme:  -->
+
+This resource supports upsert, so you can batch upsert routes on the PUT request. POST only supports single route creation.
+
+</div>
+</div>
+</div>
+
+If you don’t add a route for a certain type of entity / entities, we will default to the BigCommerce storefront URL instead.
+
+For a full list of supported `type` values, see the [Sites & Routes API](#) documentation
+
+<a href='embedded-checkout_embed-checkout' aria-hidden='true' class='block-anchor'  id='embedded-checkout_embed-checkout'><i aria-hidden='true' class='linkify icon'></i></a>
+
+## 6: Embed Checkout
+
+To generate a cart URL and set this cart as the active cart for a shopper, make a POST to:
+`https://api.bigcommerce.com/stores/{{store_hash}}/v3/carts/{{cart_id}}/redirect_urls`
+
+Once you generate the redirect URLs they can only be used once. 
+<!--
+title: "Response"
+subtitle: ""
+lineNumbers: true
+-->
+
+**Response**
+
+```js
+{
+  "cart_url": "https://store-id30h7ohwf.mybigcommerce.com/cart.php?action=load&id=bc218c65-7a32-4ab7-8082-68730c074d02&token=aa958e2b7922035bf3339215d95d145ebd9193deb36ae847caa780aa2e003e4b",
+  "checkout_url": "https://store-id30h7ohwf.mybigcommerce.com/cart.php?action=loadInCheckout&id=bc218c65-7a32-4ab7-8082-68730c074d02&token=aa958e2b7922035bf3339215d95d145ebd9193deb36ae847caa780aa2e003e4b",
+  "embedded_checkout_url": "https://store-id30h7ohwf.mybigcommerce.com/cart.php?embedded=1&action=loadInCheckout&id=bc218c65-7a32-4ab7-8082-68730c074d02&token=aa958e2b7922035bf3339215d95d145ebd9193deb36ae847caa780aa2e003e4b"
+}
+
+```
+
+Use the `embedded_checkout_url` that is returned and assemble a JSON object that will be used by the Checkout JS SDK to determine how to render the checkout.
```

Comment:
```markdown
assemble a JSON object
```

should be:

```markdown
Use the  `embedded_checkout_url` returned in the response body in the [EmbeddedCheckoutOptions](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/embeddedcheckoutoptions.md) object passed to [embedCheckout](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/README.md#embedcheckout):

\```js
embedCheckout({
"containerId": "bc-embedded-checkout",
"url": "https://store-id30h7ohwf.mybigcommerce.com/cart.php?embedded=1&action=loadInCheckout&id=bc218c65-7a32-4ab7-8082-68730c074d02&token=aa958e2b7922035bf3339215d95d145ebd9193deb36ae847caa780aa2e003e4b",
"styles": {
    "button": {
	"backgroundColor": "#ffffff",
	"borderColor": "#000000",
	"color": "#000000"
        }
    }
});
\```
```